style.css

```diff
@@ -0,0 +1,216 @@
+/* Global Style */
+* {
+  margin: 0;
+  width: 99%;
+  background-image: url('./Image/Overlay.svg');
+  font-style: normal;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+/* Log */
+.log {
+  width: 85%;
+  margin-left: 22px;
+  text-decoration: none;
+  font-size: 18px;
+  font-weight: bolder;
+  text-align: left;
+  color: #9390ff;
+}
+
+/* Icon-navigation */
+.icon-navigation {
+  font-size: 25px;
+}
+
+/* Header and item navigation */
+.top-navigation {
+  width: 95%;
+  height: 15vh;
+  margin-top: 1%;
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+}
+
+/* Item navigation */
+.item-navigation {
+  display: flex;
+  flex-direction: row;
+  flex: 2;
+  gap: 2px;
+}
+
+/* Image of circle */
+.image-circle {
+  width: 10%;
+  height: 15vh;
+  margin-left: 0%;
+}
+
+/* Section container */
+.container-section {
+  display: flex;
+  flex: row;
+}
+
+/* header container */
+.container-header {
+  width: 98.9%;
+  align-items: center;
+}
+
+/* Section Header */
+.header-text {
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 72px;
+  color: #9391ff;
+  margin-top: 2%;
+  text-align: center;
+}
+
+/* Support text of header */
+.support-text {
+  font-family: 'Roboto', sans-serif;
+  font-weight: lighter;
+  font-size: 32px;
+  line-height: 64px;
+  text-align: center;
+}
+
+/* Paragraph container */
+.text-container {
+  font-size: 21px;
+  line-height: 30px;
+  color: #b3bac5;
+  width: 95%;
+  text-align: center;
+}
+
+/* Article container */
+.container-article {
+  display: flex;
+  flex: row;
+  width: 99%;
+  margin-top: 1%;
+}
+
+/* Image circle article */
+.image-circle-2 {
+  width: 20%;
+  height: 30vh;
+  margin-left: 0%;
+}
+
+/* container thematic(hr) element */
+.thematic {
+  margin-top: 5%;
+  margin-left: 15%;
+  width: 77%;
+  height: 1vh;
+  margin-bottom: 5%;
+}
+
+/* Element hr */
+hr {
+  border: 2px solid #646369;
+}
+
+/* Container for language, frame-work, skill, */
+.experience-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2;
+  margin-left: 2%;
+  justify-content: left;
+  margin-top: 2%;
+}
+
+/* List element */
+li {
+  list-style-type: none;
+  font-size: 14px;
+  line-height: 24px;
+  color: #fff;
+}
+
+/* Header element for language. frame-work, skill */
+h3 {
+  font-weight: lighter;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #9390ff;
+  margin-bottom: 15%;
+}
+
+/* Language list container */
+.language-container {
+  display: flex;
+  flex-direction: column;
+  width: 25%;
+  height: 25vh;
+}
+
+/* Frame-work list container */
+.frame-works-container {
+  display: flex;
+  flex-direction: column;
+  width: 25%;
+  height: 20vh;
+  text-align: left;
+}
+
+/* Skill list container */
+.skills-container {
+  display: flex;
+  flex-direction: column;
+  width: 28%;
+  height: 20vh;
+  text-align: left;
+  margin-bottom: 10%;
+}
```

Comment:
This section doesn't quite match the template design.

Here's your design:

<img width="243" alt="Screen Shot 2022-05-27 at 16 11 17" src="https://user-images.githubusercontent.com/82426682/170789971-f76084f1-b2ec-46fe-96f9-128a77c01d95.png">

Here's the template design:

<img width="289" alt="Screen Shot 2022-05-27 at 16 11 35" src="https://user-images.githubusercontent.com/82426682/170790012-da45fc03-4d90-4d81-a62f-77eecb284e0f.png">

You should fix the `font-family`, the `font-size`, the `margins` between each technology and the one below it, and the `margins` between the containers and the edge of the screen. The dividers are too thick as well. Kindly fix this.